app/gather/assets/apps/utils/settings.spec.jsx

```diff
@@ -41,14 +41,16 @@ describe('Settings utils', () => {
         .get('/assets-settings')
         .reply(200, {
           aether_apps: ['kernel', 'odk', 'couchdb-sync'],
-          export_max_rows_size: 10000
+          export_max_rows_size: 10000,
+          'es_consumer_url': 'http://localhost:5601/'
```

Comment:
simple is better (you don't need the quotes for the key name) `es_consumer_url: 'http://localhost:5601/'` :wink: